actions/src/main/kotlin/com/alfresco/content/actions/sheet/ProcessDefinitionsSheet.kt

```diff
@@ -83,7 +83,8 @@ class ProcessDefinitionsSheet : BottomSheetDialogFragment(), MavericksView {
 
                             val intent = Intent(
                                 requireActivity(),
-                                Class.forName("com.alfresco.content.browse.processes.ProcessDetailActivity"),
+                                Class.forName("com.alfresco.content.process.ProcessFormActivity"),
+//                                Class.forName("com.alfresco.content.browse.processes.ProcessDetailActivity"),
```

Comment:
Why did we comment this code?

Reply:
It was navigation to the previous process detail controller and we used a new controller for dynamic forms so it's commented.

Reply:
Create a "develop" branch and raise PR again for that.